lighttpd.conf

```diff
@@ -0,0 +1,46 @@
+mimetype.assign = (
+  ".html" => "text/html",
+  ".htm" => "text/html",
+  ".shtml" => "text/html",
+  ".css" => "text/css",
+  ".gif" => "image/gif",
+  ".jpeg" => "image/jpeg",
+  ".jpg" => "image/jpeg",
+  ".js" => "application/javascript",
+  ".txt" => "text/plain",
+  ".png" => "image/png",
+  ".ico" => "image/x-icon",
+  ".svg" => "image/svg+xml",
+  ".webp" => "image/webp",
+  ".json" => "application/json",
+)
+
+server.username = "www-data"
+server.groupname = "www-data"
+
+server.max-fds = 8192
```

Comment:
how did you derive this value? from [doc](https://redmine.lighttpd.net/projects/lighttpd/wiki/Server_max-fdsDetails) (which is slightly dated..), it is documented that this value defaults to 1024 in most environment, and it is mentioned 2048 might be a better limit for "busy" servers (whatever that means, specifically).

Reply:
this value is based on a plain guess. Considering this is just the webserver serving the frontend I think default value should be sufficient 👍